2021-04-25 15:33:47,930 INFO    MainThread:6380 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 15:33:47,930 INFO    MainThread:6380 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 15:33:47,930 INFO    MainThread:6380 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\Henrik_Brådland\Google Drive\Studier\Semester 10\IKT-442\wandb\run-20210425_153347-24owxu3r\logs\debug.log
2021-04-25 15:33:47,930 INFO    MainThread:6380 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\Henrik_Brådland\Google Drive\Studier\Semester 10\IKT-442\wandb\run-20210425_153347-24owxu3r\logs\debug-internal.log
2021-04-25 15:33:47,931 INFO    MainThread:6380 [wandb_init.py:init():369] calling init triggers
2021-04-25 15:33:47,932 INFO    MainThread:6380 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-25 15:33:47,933 INFO    MainThread:6380 [wandb_init.py:init():418] starting backend
2021-04-25 15:33:47,933 INFO    MainThread:6380 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-04-25 15:33:47,940 INFO    MainThread:6380 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 15:33:48,003 INFO    MainThread:6380 [backend.py:ensure_launched():127] started backend process with pid: 2616
2021-04-25 15:33:48,003 INFO    MainThread:6380 [wandb_init.py:init():423] backend started and connected
2021-04-25 15:33:48,005 INFO    MainThread:6380 [wandb_init.py:init():465] updated telemetry
2021-04-25 15:33:48,006 INFO    MainThread:6380 [wandb_init.py:init():484] communicating current version
2021-04-25 15:33:48,889 INFO    MainThread:6380 [wandb_init.py:init():489] got version response 
2021-04-25 15:33:48,890 INFO    MainThread:6380 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-25 15:33:49,159 INFO    MainThread:6380 [wandb_init.py:init():522] starting run threads in backend
2021-04-25 15:33:50,888 INFO    MainThread:6380 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 15:33:50,889 INFO    MainThread:6380 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-04-25 15:33:50,889 INFO    MainThread:6380 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-04-25 15:33:50,889 INFO    MainThread:6380 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 15:33:50,889 INFO    MainThread:6380 [wandb_init.py:init():546] run started, returning control to user process
2021-04-25 15:33:58,030 INFO    MainThread:6380 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-25 15:33:58,031 INFO    MainThread:6380 [wandb_run.py:_restore():1466] restore
2021-04-25 15:33:58,299 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 585
  total_bytes: 585
}

2021-04-25 15:33:59,267 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 585
  total_bytes: 2805
}

2021-04-25 15:33:59,658 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2805
  total_bytes: 2805
}

2021-04-25 15:33:59,763 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2805
  total_bytes: 2805
}

2021-04-25 15:33:59,872 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2805
  total_bytes: 2805
}

2021-04-25 15:33:59,983 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2805
  total_bytes: 2805
}

2021-04-25 15:34:00,093 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2805
  total_bytes: 2805
}

2021-04-25 15:34:00,197 INFO    MainThread:6380 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2805
  total_bytes: 2805
}

2021-04-25 15:34:01,466 INFO    MainThread:6380 [wandb_run.py:_show_files():1838] logging synced files
